xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -597,7 +597,7 @@ void CGUIDialogVideoInfo::Play(bool resume)
     return;
   }
 
-  CFileItem movie(*m_movieItem->GetVideoInfoTag());
+  CFileItem movie(*m_movieItem);
```

Comment:
sounds safer. but current code is non-intuitive so if it can be changed that is preferable.